tcf_website/static/css/site/pages/landing.css

```
@@ -173,6 +173,150 @@
+.trending-card__name--compact {
+  display: block;
+  line-height: 1.2;
+}
+
+.trending-card__title {
+  display: block; /* Ensures it sits on its own line */
+  font-size: var(--text-xs);
+  opacity: 0.7;
+  margin-top: var(--space-1);
+
+  white-space: nowrap;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)